tests/transformer/test_parallel_mlp.py

```diff
@@ -19,28 +21,27 @@ def test_constructor(self, mlp):
         num_weights = sum([p.numel() for p in mlp.parameters()])
         assert num_weights == 1212
 
-    def test_cpu_forward(self, mlp):
+    def test_cpu_forward(self, mlp, transformer_config):
         # [sequence length, micro batch size, hidden size]
-        hidden_states = torch.ones((32, 2, mlp.config.hidden_size))
+        hidden_states = torch.ones((32, 2, transformer_config.hidden_size))
```

Comment:
Please explain why `transformer_config` should be added here?

Reply:
mlp, a.k.a constructed model, doesn't have config as member. So, here transformer_config is to get the proper hidden_size.

Reply:
That makes sense. Is `tranformer_config` a pytest fixture?

Reply:
yes. https://github.com/microsoft/Megatron-DeepSpeed/blob/main/tests/conftest.py#L21